lib/puppet/network/format_support.rb

```diff
@@ -11,19 +11,19 @@ module ClassMethods
     def convert_from(format, data)
       get_format(format).intern(self, data)
     rescue => err
-      raise Puppet::Network::FormatHandler::FormatError, "Could not intern from #{format}: #{err}", err.backtrace
+      raise Puppet::Network::FormatHandler::FormatError, _("Could not intern from %{format}: %{err}") % { format: format, err: err }, err.backtrace
```

Comment:
All the translated strings in this file (format_support.rb) have the same keyword problem, which is they are descriptive names for functions, but technically they are keywords. Should they be left out of translations?